libafl_frida/src/coverage_rt.rs

```diff
@@ -51,8 +75,65 @@ impl FridaRuntime for CoverageRuntime {
 
     fn post_exec<I: libafl::inputs::Input + libafl::inputs::HasTargetBytes>(
         &mut self,
-        _input: &I,
+        input: &I,
     ) -> Result<(), libafl::Error> {
+        if self.save_dr_cov {
+            self.cnt += 1;
+            if self.max_cnt > 0 && self.cnt < self.max_cnt {
+                return Ok(());
+            }
+
+            // Create basic blocks
+            let mut drcov_basic_blocks: Vec<DrCovBasicBlock> = vec![];
+
+            for (key, value) in &self.basic_blocks {
+                // Is map[key] greater than 0?
+                if self.inner_bbs.borrow().map[*key as usize] == 0 {
+                    continue;
+                }
+
+                log::trace!(
+                    "Covered BB key: {:x}, value: {:016x} - {:016x}",
+                    key,
+                    value.start,
+                    value.end
+                );
+                drcov_basic_blocks.push(*value);
+            }
+
+            let mut input_hasher = RandomState::with_seeds(0, 0, 0, 0).build_hasher();
```

Comment:
Perhaps get the name from the Input directly, instead of rehashing it and possibly using a different name?

Reply:
Fixed